src/DacpacTool/PackageBuilder.cs

```diff
@@ -401,7 +401,9 @@ public void GenerateCreateScript(FileInfo dacpacFile, string databaseName)
 
             using var package = DacPackage.Load(dacpacFile.FullName);
             using var file = File.Create(Path.Combine(dacpacFile.DirectoryName, scriptFileName));
-            DacServices.GenerateCreateScript(file, package, databaseName);
+
+            var options = new DacDeployOptions() { IncludeCompositeObjects = true };
```

Comment:
It looks like we're hardcoding this to be true now which might not be what everyone wants. Ideally we'd pass the value of the MSBuild property down to here and use that.

Reply:
Yes, you're right. I changed this,